service-desk.md

```diff
@@ -9,13 +9,12 @@ The individuals who currently have access are listed below.
 
 | Name | GitHub Profile |
 | ---- | ------- |
-| Aaron Crickenberger | **[@spiffxp](https://github.com/spiffxp)** |
```

Comment:
It would be helpful to have access to this as a wg-k8s-infra organizer, though I have overlap available via @dims for now

Reply:
@spiffxp can you get an issue created to track this?